### PR TITLE
Add additional integration tests

### PR DESCRIPTION
**Pull request by jasdel**
Adds additional integration tests for API clients.

### CODE REVIEW

File: internal/integrationtest/go.mod [hunk 1]
```diff
@@ -1,11 +1,25 @@
 module github.com/aws/aws-sdk-go-v2/internal/integrationtest
 
 require (
+	github.com/aws/aws-sdk-go v1.34.32
```
Comment (skotambkar) [inline internal/integrationtest/go.mod:4]: Incorrect dep

**Review by skotambkar — COMMENTED** [review]
Needs rebase

File: internal/integrationtest/go.mod (hunk 1 above)
Comment (jasdel) [reply internal/integrationtest/go.mod:4]: Thanks!

**Review by skotambkar — APPROVED** [review]

**Review by skotambkar — APPROVED** [review]

File: internal/integrationtest/apigateway/smoke_test.go [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // +build integration
 
-package apigateway
+package directconnect
```
Comment (skotambkar) [inline internal/integrationtest/apigateway/smoke_test.go:3]: A little accident here I suppose.

Probably just need to update the file name to directconnect/smoke_test.go

**Review by skotambkar — CHANGES_REQUESTED** [review]

**Review by skotambkar — APPROVED** [review]

**Pull request merged**